core/server/services/auth/api-key/content.js

```diff
@@ -0,0 +1,46 @@
+const models = require('../../../models');
+const {
+    BadRequestError,
+    UnauthorizedError
+} = require('../../../lib/common/errors');
+
+const authenticateContentApiKey = function authenticateContentApiKey(req, res, next) {
+    if (req.headers && req.headers.authorization) {
```

Comment:
Same as with the admin middleware, if we maybe have a `rejectAdminKey` and `rejectContentKey` middleware :man_shrugging: or something of that ilk

Reply:
👍 sounds like a good plan